bindings/nodejs/lib/gpio.js

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
The location of this file is actually kinda problematic. If you want to use soletta GPIO, you now have to ```require( "soletta/bindings/nodejs/lib/gpio" );``` in your project.

Ideally this should be ```require( "soletta/gpio" );```, but that means you would have to move gpio.js to the repository root, which might pollute it.